pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/WheelUtils.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.linkedin.gradle.python.util;
+
+import com.linkedin.gradle.python.extension.PythonDetails;
+import com.linkedin.gradle.python.plugin.PythonHelpers;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import org.apache.commons.io.FileUtils;
+import org.gradle.api.Project;
+import org.gradle.api.logging.Logger;
+import org.gradle.api.logging.Logging;
+import org.gradle.process.ExecResult;
+
+
+public class WheelUtils {
```

Comment:
The class is a utility class in the sense that it provides some functions (static methods in absence of better in Java), but we don't need to call it `Utils`. Do we plan to add other functions here? If yes, perhaps a full name `WheelUtilities` would be nicer. If not, perhaps just its current purpose should do. Something like `WheelMaker` perhaps?